### PR TITLE
[GSW-371] fix: Manage slippage on local storage

### PR DESCRIPTION
**Pull request by jinoosss**

### CODE REVIEW

File: packages/web/src/hooks/common/use-slippage.ts [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+import { useCallback } from "react";
+import { toDecimalNumber } from "@utils/number-utils";
+import { DEFAULT_SLIPPAGE } from "@constants/option.constant";
+
+export const useSlippage = () => {
+  const [slippage, setSlippage] = useAtom(CommonState.slippage);
+
+  const changeSlippage = useCallback(
+    (slippage: number) => {
+      let changedSlippage = slippage;
```
Comment (harryoh) [inline packages/web/src/hooks/common/use-slippage.ts:12]: How about it?
```js
const changedSlippage = Math.min(100, Math.max(0, parseInt(slippage)));
```

File: packages/web/src/hooks/common/use-slippage.ts [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+      if (slippage < 0) {
+        changedSlippage = 0;
+      } else if (slippage > 100) {
+        changedSlippage = 100;
+      }
+      changedSlippage = toDecimalNumber(changedSlippage, 2);
```
Comment (harryoh) [inline packages/web/src/hooks/common/use-slippage.ts:18]: Does `slippage` need to `bignumber`?

Comment (jinoosss) [reply packages/web/src/hooks/common/use-slippage.ts:18]: No, I wanted to round to the second decimal place.
But this seems to be wrong.

**Review by harryoh — APPROVED** [review]
👍

**Pull request merged**